diyspk/lsscsi/Makefile

```diff
@@ -0,0 +1,20 @@
+SPK_NAME = lsscsi
+SPK_VERS = 0.30
+SPK_REV = 1
+
+DEPENDS = cross/lsscsi
+
+MAINTAINER = biyiklioglu
+DESCRIPTION = The lsscsi command lists information about SCSI devices in Linux.
+DISPLAY_NAME = lsscsi
+STARTABLE = no
+
+HOMEPAGE   = http://sg.danny.cz/scsi/lsscsi.html
+LICENSE    = MIT
+HELP_URL   = http://sg.danny.cz/scsi/lsscsi.html
+
+SERVICE_SETUP = src/service-setup.sh
+
+INSTALL_PREFIX = /usr/local/$(SPK_NAME)
```

Comment:
Default value points to `/var/packages/${SYNOPKG_PKGNAME}/target`. Please do not override.